app/tests/spec/views/sign_up.js

```diff
@@ -229,6 +248,38 @@ define(function (require, exports, module) {
             });
         });
       });
+
+      describe('password confirm', function() {
+        it('is visible if enabled', function () {
+          sinon.stub(experimentGroupingRules, 'choose', (name) => {
+            return name === 'signupPasswordConfirm' && 'treatment';
+          });
+
+          delete view._passwordConfirmExperiment;
+          enableExperiments();
+
+          return view.render()
+            .then(function () {
+              assert.isTrue(experimentGroupingRules.choose.calledWith('signupPasswordConfirm'));
+              assert.equal(view.$('#vpassword').length, 1);
```

Comment:
Oh! I just copied what was right above this, for the email-marketing-optin length check. It seems to be that the `lengthOf` way is 4 characters less...